env.sample

```diff
@@ -1,4 +1,6 @@
- NETWORK=
- PROVIDER_URL=
- AINIZE_INTERNAL_PRIVATE_KEY=
- BOT_NAME=
\ No newline at end of file
+NETWORK=
```

Comment:
I think the NETWORK constant is related to the PROVIDER_URL. It seems that we can find the NETWORK value by referring to the PROVIDER_URL value at runtime without having to set both.

Reply:
+) we need to set PROVIDER_URL and AINIZE_INTERNAL_PRIVATE_KEY manually in local environment. To build the same environment as Ainize Container